app/services/submit_to_app_store.rb

```diff
@@ -31,7 +31,12 @@ def submit(submission)
     # able to support re-sending/submitting an appplication so we can ignore
     # put requests
     post_manager = AppStoreClient.new
-    post_manager.post(payload)
+
+    if submission.is_a?(PriorAuthorityApplication)
```

Comment:
You'll also need to look in SubmitToAppStore::PriorAuthority::EventBuilder because there are some TODO's to actually populate the event with the latest further_information data.